app/main.py

```diff
@@ -2,5 +2,18 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+
+    results = []
+
+    def wrapper(*args) -> any:
+        for result in results:
+            if result["func"] == func and result["args"] == args:
+                print("Getting from cache")
+                return result["output"]
```

Comment:
no need to use loop

Reply:
just write args as the key in the dictionary, and the value is the result of the function